tests/e2e/src/pages/main.page.ts

```diff
@@ -162,6 +170,20 @@ export class MainPage extends BasePage {
     await this.verifyElement("xpath", selector, checkType);
   }
 
+  async checkArrowElement(externalLinkName: string, checkType: string) {
```

Comment:
@amelnytskyi please discuss this step with Anton, because I still think it will be better to extract all links in cucumber into some Scenario Outline table. You can explain to him your idea of supporting this method and a case with extending of the existing one by (uniq text and href are visible\clickable) in async returnElementByType(elementType: string, value: string) {}